config/tasklists/learn-to-drive-a-car.json

```diff
@@ -1,158 +0,0 @@
-              "text": "Practise vehicle safety questions"
-            }
-          ]
-        },
-        {
-          "title": "Prepare for your theory test",
-          "panel_links": [
-            {
-              "href": "/theory-test/revision-and-practice",
-              "text": "Theory test revision and practice"
-            },
-            {
-              "href": "/take-practice-theory-test",
-              "text": "Take a practice theory test"
-            },
-            {
-              "href": "https://www.safedrivingforlife.info/shop/product/official-dvsa-theory-test-kit-app-app",
-              "text": "Theory and hazard perception test app"
-            }
-          ]
-        }
-      ],
-      [
-        {
-          "title": "Book and manage your theory test",
-          "panel_descriptions": ["You need a provisional driving licence to book your theory test."],
-          "panel_links": [
-            {
-              "href": "/book-theory-test",
-              "text": "Book your theory test"
-            },
-            {
-              "href": "/theory-test/what-to-take",
-              "text": "What to take to your test"
-            },
-            {
-              "href": "/change-theory-test",
-              "text": "Change your theory test appointment"
-            },
-            {
-              "href": "/check-theory-test",
-              "text": "Check your theory test appointment details"
-            },
-            {
-              "href": "/cancel-theory-test",
-              "text": "Cancel your theory test"
-            }
-          ]
-        }
-      ],
-      [
-        {
-          "title": "Book and manage your driving test",
-          "panel_descriptions": ["You must pass your theory test before you can book your driving test."],
-          "panel_links": [
-            {
-              "href": "/book-driving-test",
-              "text": "Book your driving test"
-            },
-            {
-              "href": "/driving-test/what-to-take",
-              "text": "What to take to your test"
-            },
-            {
-              "href": "/change-driving-test",
-              "text": "Change your driving test appointment"
-            },
-            {
-              "href": "/check-driving-test",
-              "text": "Check your driving test appointment details"
-            },
-            {
-              "href": "/cancel-driving-test",
-              "text": "Cancel your driving test"
-            }
-          ]
-        }
-      ],
-      [
-        {
-          "title": "When you pass",
-          "panel_descriptions": ["You can start driving as soon as you pass your driving test.","You must have an insurance policy that allows you to drive without supervision."],
-          "panel_links": [
-            {
-              "href": "/pass-plus",
-              "text": "Find out about Pass Plus training courses"
-            }
-          ]
-        }
-      ]
-    ]
-  }
-}
```

Comment:
Presumably this has been removed because it's now coming from the navigation helpers gem?

Reply:
👍

Reply:
I'll update the commit message to reflect